src/test/resources/30/expanded/valid/fxl-svg-width-height/EPUB/cover.svg

```diff
@@ -0,0 +1,262 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
A more minimal SVG would be helpful for easier readability and maintainability.

We could use sth like the following as a base document:

```svg
<?xml version="1.0" encoding="utf-8"?>
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="100%" height="100%"
  viewBox="0 0 100 100">
  <desc>Test SVG</desc>

  <rect x="10" y="10" width="80" height="80" fill="none" stroke="blue" stroke-width="2"/>
</svg>
```
(in this specific test sample, we would remove the `viewBox` attribute).

Reply:
Also, maybe the test file name 'fxl-svg-no-viewbox' would be more explicit?